src/app/zap-templates/zcl/data-model/chip/operational-credentials-cluster.xml

```diff
@@ -113,9 +113,9 @@ limitations under the License.
 
     <command source="server" code="0x08" name="NOCResponse" optional="false">
       <description>Response to AddNOC or UpdateNOC commands.</description>
-      <arg name="StatusCode" type="INT8U"/>
-      <arg name="FabricIndex" type="INT8U"/>
-      <arg name="DebugText" type="CHAR_STRING"/>
+      <arg name="StatusCode" type="OperationalCertStatus"/>
+      <arg name="FabricIndex" type="INT8U" optional="true"/>
```

Comment:
This should be of type `type="fabric_idx"`

Reply:
Good catch, thank you.  Fixed it in RemoveFabric as well.